### DISCUSSION TITLE
Urgent! PLEASE HELP!  After website move getting 404 error message on meeting list URL  and meeting list not displaying

### DISCUSSION THREAD

**Discussion by JoeBryan7873** [code4recovery/12-step-meeting-list#1137] [Q&A] [answered]
Hi,

I am having an urgent issue. We just update dour website an after moving from a development server the 12 step meeting list plugin is not working.

I uninstalled and reinstalled the plugin and re-imported the meeting data I'm getting a 404 error for the meeting list URL
https://capecodaa.net/meetings/

I really need this fixed quickly
Please help!

**Comment by joshreisner** [comment]
hi @JoeBryan7873 go to wordpress settings > permalinks and hit "save changes" and it should come back

**Reply by JoeBryan7873** [reply, marked as answer]
It worked! Thank you! You are a life saver

**Reply by joshreisner** [reply]
for some extra background, we have a similar issue with some sites currently getting permalinks reset late at night (we think by woocommerce, maybe) but this has always been a pretty common (and unfortunate) effect of moving a wordpress site between environments. this affects other plugins as well: https://theeventscalendar.com/knowledgebase/k/fixing-http-404-errors/
